README.md

```diff
@@ -199,6 +199,29 @@ Flags:
   * --s: The service ID to download. Current working directory is assumed.
   * --v: The version to download. The currently active version is assumed.
 
+### logging 
```

Comment:
@stephenbasile updated README, can you review pls ?

Reply:
Just noting our discussion in slack in case we forget: Let's get the flags documented in the README then we are good to go.

Reply:
README updated, let me know if format is OK.  made status, bigquery subs of Logging , since logging is the parent command so we can easily add more subcommands there.

Reply:
👍

Reply:
OK everything is good to go, ready to merge if you are good !